ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/slider/SliderScreen.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.android.wearable.composeadvanced.presentation.ui.slider
```

Comment:
Missing open source header.

Reply:
I also added the open source header.

Reply:
tip: Setup this in Android Studio

![image](https://user-images.githubusercontent.com/231923/154538808-5b5e68e1-501e-4771-9f78-0e30da369d42.png)